pkg/registry/constants.go

```diff
@@ -20,11 +20,14 @@ const (
 	// HelmChartDefaultTag is the default tag used when storing a chart reference with no tag
 	HelmChartDefaultTag = "latest"
 
-	// HelmChartMetaMediaType is the reserved media type for Helm chart metadata
-	HelmChartMetaMediaType = "application/vnd.cncf.helm.chart.meta.v1+json"
+	// HelmChartConfigMediaType is the reserved media type for the Helm chart manifest config
+	HelmChartConfigMediaType = "application/vnd.cncf.helm.chart.config.v1+json"
```

Comment:
is there a difference in intent between `meta` and `config` in terms of OCI usage here? Why is the name being changed?

Reply:
config is the object used to differentiate the `artifactType`. 
In lieu of the pending distribution PR, here's a quick guide for types: https://stevelasker.blog/2019/05/11/authoring-oci-registry-artifacts-quick-guide/
Artifact Type Format
Artifacts are defined by `manifest.config.mediaType` using the following format:

`application/vnd.`[org/company]`.`[objectType]`.`[optional sub type]`.`config.`[version]`+json

Reply:
I don't understand *why* we're re-naming it to `config`, and from what I read in that blog post I could not figure out the reason. Can you point out *why* we need to change it to `config`? `meta` is the standard descriptor we've been using for chart metadata so I'm not sure why we need to change the convention to follow.

The [image-spec documentation](https://github.com/opencontainers/image-spec/blob/master/media-types.md) doesn't appear to follow that convention laid out in the blog post either. `manifest`, `descriptor`, `index`, `header` and others all don't follow that convention, so I'm led to believe that this isn't a convention we need to follow.

According to the image-spec, the [OCI image config specification](https://github.com/opencontainers/image-spec/blob/master/config.md) uses the term `config` because the metadata stored in that object is the image configuration - that is, it describes how the layers should be used by a container runtime and execution tool and its relationship to filesystem changesets, and therefore it's "config". In other words, the content and how it's being used defines the schema, not the other way around.

Reply:
The initial design was to actually have an artifact type property as a sibling to .config. This was pushed out since landing V1 was priority. 
If we did get artifactType in V2 then the config for helm would actually truly be config metadata. Either way works.